rust/release-checklist.md

```diff
@@ -38,19 +38,30 @@ Push access to the upstream repository is required in order to publish the new t
 
 :warning:: if `origin` is not the name of the locally configured remote that points to the upstream git repository (i.e. `git@github.com:coreos/{{ git_repo }}.git`), be sure to assign the correct remote name to the `UPSTREAM_REMOTE` variable.
 
+- declare environment variables
+  - [ ] `RELEASE_VER=x.y.z`
+  - [ ] `UPSTREAM_REMOTE=origin`
+
+- update all dependencies
+  - [ ] `git checkout -b pre-release-${RELEASE_VER}`
+  - [ ] `cargo update`
+  - [ ] `git commit -m "cargo: update dependencies"`
```

Comment:
We need to either use `--all` or specify the files to add.  We haven't cleaned the working directory yet, so the latter is safer:

```suggestion
  - [ ] `git add Cargo.lock && git commit -m "cargo: update dependencies"`
```

Reply:
Thank you, I had not considered that being a possible area for unwanted changes.